packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -27,16 +27,17 @@ import {
   statusDescriptionTagName,
 } from './transform';
 import { OT_REQUEST_HEADER } from './utils';
+import { SERVICE_RESOURCE } from '@opentelemetry/resources';
 /**
  * Zipkin Exporter
  */
 export class ZipkinExporter implements SpanExporter {
   static readonly DEFAULT_URL = 'http://localhost:9411/api/v2/spans';
   private readonly _logger: api.Logger;
-  private readonly _serviceName: string;
   private readonly _statusCodeTagName: string;
   private readonly _statusDescriptionTagName: string;
   private readonly _reqOpts: http.RequestOptions;
+  private _serviceName?: string;
   private _isShutdown: boolean;
 
   constructor(config: zipkinTypes.ExporterConfig) {
```

Comment:
Since all configs are now optional, we can make the whole config object optional 
```suggestion
  constructor(config: zipkinTypes.ExporterConfig = {}) {
```